scripts/footprint/size_report

```diff
@@ -56,6 +56,7 @@ def load_symbols_and_paths(bin_nm, elf_file, path_to_strip=None):
     nm_out = subprocess.check_output(
         [bin_nm, elf_file, "-S", "-l", "--size-sort", "--radix=d"])
     for line in nm_out.decode('utf8').split('\n'):
+        line = line.strip()
```

Comment:
Alternatively, line.replace('\r', '') can be used.

Reply:
Or just pass `universal_newlines=True` to `subprocess.check_output()`. This [should make things work everywhere](https://docs.python.org/3/glossary.html#term-universal-newlines).

Reply:
Agreed, @lpereira's suggestion is the cleanest.